sdk/loadtestservice/Azure.ResourceManager.LoadTesting/src/Generated/Models/LoadTestResourcePageList.Serialization.cs

```diff
@@ -16,7 +16,7 @@ internal partial class LoadTestResourcePageList
     {
         internal static LoadTestResourcePageList DeserializeLoadTestResourcePageList(JsonElement element)
```

Comment:
Should the suffix be PageList or ListResult? There are other places where ListResult is used as suffix for similar concept eg. LoadTestingQuotaListResult. The suffix should be consistent?

Reply:
This is internal function that don't need to change the name